app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
+
+    @classmethod
+    def pick_by_name(cls, in_name: str) -> object:
+        for person in Person.people:
+            if person == in_name:
+                return Person.people[person]
```

Comment:
You don't need to use a loop here, because you spend extra time for it every time you call this method. You can get the value from the dict with the name as a key.

Reply:
yes)))) thanks))